01_fib/shahriys/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var outfib io.Writer = os.Stdout
+
+// Function to printh first n Fibonacci or Negafibonacci numbers
+func fib(n int) {
+	switch n {
+	case 0:
+		fmt.Fprintln(outfib, 1)
+	default:
+		if n > 0 {
+			n1, n2 := 1, 1
+			for i := 0; i < n; i++ {
+				switch i {
+				case 0, 1:
```

Comment:
You can actually remove this switch statement by changing the code in the `default` block slightly

Reply:
branch Fixed.